src/requesthandler/RequestHandler_Config.cpp

```diff
@@ -290,7 +290,7 @@ RequestResult RequestHandler::CreateProfile(const Request &request)
 		return RequestResult::Error(RequestStatus::ResourceAlreadyExists);
 
 	QMainWindow *mainWindow = static_cast<QMainWindow *>(obs_frontend_get_main_window());
-	QMetaObject::invokeMethod(mainWindow, "NewProfile", Qt::BlockingQueuedConnection,
+	QMetaObject::invokeMethod(mainWindow, "CreateNewProfile", Qt::BlockingQueuedConnection,
```

Comment:
Plugins are not supposed to invoke private API methods directly - the method names can change at any point (as just happened) and the existence of these functions is not something plugins should know about.

Thus this isn’t a ‘fix’, but instead perpetuates the same mistake that lead to the bug in the first place.

As pointed out in the OP, this should use the front end API instead, which was updated to call the correct (updated) function internally and thus provides the stable interface for plugins.

Reply:
It's outside my time and ability to file that PR but a simple function name swap was simple enough.

A competiting PR is needed doing it "properly"

Reply:
And theres a possibly other functions across the websocket plugin (such as `DeleteProfile` that "do it wrong" too, seems a further/in depth audit and revise needed?)

Reply:
I think that should do it?